content/micropython/01.basics/07.installing-modules/installing-modules.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Installing Modules
+description: Learn how to install and remove external modules on your Arduino board.
+author: Karl Söderby
+micropython_type: basics
+hero_image: "./hero-banner.png"
+---
+
+Your MicroPython installation comes with a set of modules built-in to the firmware. If you are using any external components such as sensors, displays or neopixels, you will need to install these manually.

```

Comment:
Hmm I am uncertain about this, because the terminology seems a bit contested at the moment:
- Module - mainly pointing towards a specific file, i.e.`module.py`,
- Library - can be a _collection_ of modules. E.g. the cloud library bundles a bunch of different modules.

So for now, let's leave it as "modules", just because it is Python.

Reply:
Fair enough, maybe library is not the right comparison. 
But I feel like there is something missing explaining what a module is, and why I want to install one.